bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        - alert: HighClientGoRequestLatency
+          annotations:
+            summary: "Client-go requests latency are too high on {{ $labels.url_host }}"
+            description: "{{ $labels.url_host }} has a median request latency above 500ms (current value: {{ $value }}s labels: {{ $labels }})"
```

Comment:
@dgrisonnet can you confirm if my understanding of the alert templating is right?

Reply:
yes, but I am not sure about `$labels`. I think you need to specify each label name manually, but I might be wrong here.

Reply:
we'll discuss alerting in another PR